packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -128,31 +147,49 @@ export const companyNamesAndUrls: TcompanyNamesAndUrls = {
 };
 
 export const AppToContentMapper: TAppToContentMapper = {
-    ctrader: {
+    ctrader_web: {
```

Comment:
@hirad-deriv we need to push it back to BE, we should not hardcode these links here.Please raise it in the channel and ask @dillys do create a card for that.

Reply:
I am following up on these matters. These will be added once we receive the new API of merged platforms.